setup.py

```diff
@@ -196,7 +196,10 @@ def get_long_description():
     install_requires=["numpy"],
     extras_require={"doc": ["numpydoc", "sphinx", "gitpython"]},
     cmdclass=cmdclass,
-    setup_requires=["numpy"],
+    setup_requires=[
+        "oldest-supported-numpy ; python_version < '3.9'",
```

Comment:
I think `numpy` is enough, isn't it? No need to require `oldest-supported-numpy` at runtime. Building against it will enable backwards compatibility, won't it?